openssl/src/ssl/mod.rs

```diff
@@ -3083,6 +3093,39 @@ impl SslRef {
         }
     }
 
+    /// Decodes a slice of wire-format cipher suite specification bytes.
+    ///
+    /// Returns a tuple of ciphers and signalling cipher-suite values if valid, or `None` otherwise.
+    ///
+    /// Requires OpenSSL 1.1.1 or newer.
+    #[corresponds(SSL_bytes_to_cipher_list)]
+    #[cfg(ossl111)]
+    pub fn bytes_to_ciphers_stack(
+        &self,
+        bytes: &[u8],
+        isv2format: bool,
+    ) -> Option<(Stack<SslCipher>, Stack<SslCipher>)> {
```

Comment:
Can we have this return a `struct CipherLists { pub suites: Stack<SslCipher>, pub signaling_suites: Stack<SslCipher> }` so it's a bit more descriptive?

Reply:
Does the function not push errors onto the stack on failure, or should this return a `Result<CipherLists, ErrorStack>` instead?